src/levanter/layers/attention.py

```diff
@@ -1462,6 +1463,9 @@ class AttentionConfig:
     """Configuration for QK normalization. If None, no normalization is applied."""
 
     def __post_init__(self):
+        if self.use_mup and self.scaling_factor is None:
+            object.__setattr__(self, "scaling_factor", 1 / self.head_size)
```

Comment:
i'd ratehr set this in `init` and not `__post_init__` (since I'd prefer to avoid setattr unless we have ot